Project.toml

```diff
@@ -11,13 +11,13 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Meshes = "eacbb407-ea5a-433e-ab97-5258b1ca43fa"
 PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
```

Comment:
I previously placed SciMLBase in the "deps" section because there was a bug when placing it in the "weakdeps" section, but I did not find a solution.

Reply:
I had the same problem as well.

Reply:
I feel that the documentation for the `extension` is not very complete, and many details need to be checked in those issues and forum posts.